manual/guides/cyclus.md

```diff
@@ -0,0 +1,71 @@
+---
+layout: manual
+title: Cyclus
+subtitle: "Cyclus - Creating your Ubuntu Environment"
+permalink: /manual/guides/cyclus
+---
+
+## Step 0. Clean Install Ubuntu
+Make sure your Ubuntu environment is clean to ensure the script works.
```

Comment:
Be clear about what you mean here.... (how would someone check for a clean environment?) 
(e.g. delete any old cyclus installations, check bashrc, etc.)